data/coalition/coalition missions.txt

```diff
@@ -1565,58 +1558,52 @@ mission "Saryd University Lecture"
+					goto hai
+					to display
+						has "First Contact: Hai: offered"
 				`	"A few humans know about the Hai, but there's no official acknowledgment of their existence. They live to the north of human space, and we can visit them through a wormhole."`
-					goto "know"
+					goto hai
+					to display
+						has "First Contact: Hai: offered"
+						not "event: hai-human resolution announced"
 				`	"Humans have official contact with the Quarg, but the Quarg don't trade technology with humanity."`
 				`	"Humans have official contact with the Quarg, but humans and Quarg don't really intermix."`
 				`	"Humans know about the Quarg, but most people know little about them."`
-			label quarg
 			`	The audience response is muted. You get the impression that you haven't told them anything they don't already know.`
 				goto nextquestion
-			label know
-			branch coalitionknow
-				has "coalition knows hai"
+			label hai
 			action
 				set "coalition knows hai"
 			`	Several members of the audience are delighted. "Unknown to us, this species is," says the Heliarch. "That you have made contact with them despite the Quarg, good news it is."`
-			`	"Tell us more of them, could you?" asks another scientist.`
-			`	As you open your mouth to speak, the Heliarch cuts you off. "Enough questions on the Hai, we have." The crowd looks disappointed at this development.`
-				goto nextquestion
-			label coalitionknow
+				to display
+					not "coalition knows hai"
 			`	Several members of the audience are delighted. "Ah, the Hai, we have heard of," says the Heliarch. "That you have made contact with them despite the Quarg, good news it is."`
+				to display
+					has "coalition knows hai"
 			`	"Tell us more of them, could you?" asks another scientist.`
 			`	As you open your mouth to speak, the Heliarch cuts you off. "Enough questions on the Hai, we have." The crowd looks disappointed at this development.`
 			label nextquestion
-			branch war
-				has "FWC End: completed"
 			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
 			choice
 				`	"No, human space is split between the Republic in the north and the Free Worlds in the south."`
+					to display
+						not "FWC End: completed"
 				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they have a lot of autonomy."`
-			`	As you answer the question, your translator begins to cut out. Repeating your answer does no good, as it continues to fizzle out as you talk. You look over to the Heliarch, who eyes the curious Kimek with suspicion. "Apologize for the malfunction, I must," says the Heliarch. "The next question, answer it please."`
-				goto questionthree
-			label war
-			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
-			choice
+					to display
+						not "FWC End: completed"
```

Comment:
Should this only appear if the player chose the Reconciliation path?

Reply:
I believe the distinction is made because, at the end of the checkmate arc, the FW is said to still pay some small tax to the Republic, while no reference to such a tax is made in the reconciliation path.

Reply:
I didn't recognise the condition. All good : )

Reply:
Well, I haven't actually checked this condition, I'm just assuming it's the one set at the end of FW checkmate, but for all I know, it's been the wrong condition the whole time this mission has existed.

Reply:
I checked and the condition is `"free worlds checkmate"`

Reply:
Indeed. Should be fixed now.